html/pfappserver/lib/pfappserver/Model/Roles.pm

```diff
@@ -129,6 +130,9 @@ sub delete {
     my ($self, $role_ref) = @_;
 
     my $logger = get_logger();
+    if (exists $STANDARD_ROLES{$role_ref->{name}}) {
+        return ($STATUS::FORBIDDEN, "The role $role_ref->{name} a standard role cannot be remove");
```

Comment:
- I'd put the role name between '' ie: The role 'patate' ...
- is a standard role ?
- cannot be removed ?